prov/efa/src/rxr/rxr.h

```diff
@@ -422,8 +419,8 @@ struct rxr_ep {
 	struct dlist_entry rx_entry_queued_ctrl_list;
 	/* rx entries with queued rnr packets */
 	struct dlist_entry rx_entry_queued_rnr_list;
-	/* tx_entries with data to be sent (large messages) */
-	struct dlist_entry tx_pending_list;
+	/* rx_entries with data to be sent (large messages) */
+	struct dlist_entry op_pending_list;
```

Comment:
should be named `op_entry_longcts_send_list` because all entries on it is waiting to send data.